.github/workflows/ruby-build.yml

```diff
@@ -95,7 +95,7 @@ jobs:
         uses: ./.github/actions/fetch-codeql
       - name: Build Query Pack
         run: |
-          codeql pack create ../shared/ssa --output target/packs
+          codeql pack install ql/lib
```

Comment:
Will this still build the pack from the repo, or fetch it from the registry?

Reply:
I guess it fetches it from the registry.

Reply:
That doesn't work then. We do not guarantee, that the code in `main` is compatible with any released shared pack; only that it is compatible with the version of the pack at `main` as well.

Reply:
Ah ok. In that case we should remove the `codeql pack install ql/src` call too and use the suite-helpers from `../misc`